integration-tests/README.md

```diff
@@ -26,6 +26,35 @@ cargo test -p integration-tests --jobs 1 -- --test-threads 1
 cargo test -p integration-tests --features docker-test
 ```
 
+## Logging and Tracing
+We have three types of logging available:
+
```

Comment:
Very nice to include docs! Is there a reason why it is in `integration-tests/README.md`? Is this only meant for integration tests?

Reply:
Not only. Our top level README is minimalistic, I did not wanted to add Logging sections there.
But it is a good point. I've added `Contributing` ling to the main Readme and now it points to this doc.